java/server/src/org/openqa/grid/web/servlet/beta/ConfigPrinter.java

```diff
@@ -0,0 +1,41 @@
+// Licensed to the Software Freedom Conservancy (SFC) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The SFC licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package org.openqa.grid.web.servlet.beta;
+
+import org.openqa.grid.common.RegistrationRequest;
+
+import java.util.concurrent.TimeUnit;
+
+public class ConfigPrinter {
+
+  public static String printConfigValue(String configKey, String configValue) {
```

Comment:
instead of forcing everything to a string and then back to an int, let's change the method signature to be Object configValue and then check if the Object is an integer.